### PR TITLE
[fix]: [PIE-4615]: DTOs moved out of 900-git-sync-sdk to 878-ng-common-utilities

### PR DESCRIPTION
**Pull request by senjucanon2**
## Describe your changes:
GitImportInfoDTO has to be moved out of 900-git-sync-sdk, to 878-ng-common-utilities

## Checklist
- [ ] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [x] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)

### CODE REVIEW

**Review by archit-harness — DISMISSED** [review]
> Reviewed 6 of 6 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: 6 file(s) to review. 1 of 2 approvals obtained.

File: 878-ng-common-utilities/src/main/java/io/harness/gitaware/helper/GitImportInfoDTO.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/05/PolyForm-Free-Trial-1.0.0.txt.
  */
 
-package io.harness.gitsync.interceptor;
+package io.harness.gitaware.helper;
```
Comment (namanvermah) [inline 878-ng-common-utilities/src/main/java/io/harness/gitaware/helper/GitImportInfoDTO.java:8]: We should move all the dtos that were in the same package as import info dto

**Review by namanvermah — CHANGES_REQUESTED** [review]

**Review by archit-harness — CHANGES_REQUESTED** [review]
> Reviewed 4 of 8 files at r2, 32 of 32 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: 37 file(s) to review. 2 change requests. 0 of 2 approvals obtained. 

___
*[`pipeline-service/src/main/java/io/harness/pms/ngpipeline/inputset/resources/InputSetResourcePMSImpl.java` line 28 at r1](https://reviewable.io/reviews/harness/harness-core/35825#-N87nV6oCDVrRBI18ytX:-N87nV6oCDVrRBI18ytY:b1qi62q) ([raw file](https://github.com/harness/harness-core/blob/8df8f287f57b81d6a2716327a33fee528a3c027d/pipeline-service/src/main/java/io/harness/pms/ngpipeline/inputset/resources/InputSetResourcePMSImpl.java#L28)):*
> ```Java
> import io.harness.gitsync.interceptor.GitEntityCreateInfoDTO;
> import io.harness.gitsync.interceptor.GitEntityDeleteInfoDTO;
> import io.harness.gitsync.interceptor.GitEntityFindInfoDTO;
> ```

why are they deleted? same for below files

**Review by senjucanon2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: 37 file(s) to review. 2 change requests. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`pipeline-service/src/main/java/io/harness/pms/ngpipeline/inputset/resources/InputSetResourcePMSImpl.java` line 28 at r1](https://reviewable.io/reviews/harness/harness-core/35825#-N87nV6oCDVrRBI18ytX:-N87wySB5HK7x3Lf0eFF:b-qr7w5x) ([raw file](https://github.com/harness/harness-core/blob/8df8f287f57b81d6a2716327a33fee528a3c027d/pipeline-service/src/main/java/io/harness/pms/ngpipeline/inputset/resources/InputSetResourcePMSImpl.java#L28)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

why are they deleted? same for below files
</blockquote></details>

I moved all DTO's from 900-git-sync-sdk to 878-bg-common-utilities, so upon refactoring these imports were automatically modififed

**Review by senjucanon2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: 37 file(s) to review. 2 change requests. 2. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`878-ng-common-utilities/src/main/java/io/harness/gitaware/helper/GitImportInfoDTO.java` line 8 at r2](https://reviewable.io/reviews/harness/harness-core/35825#-N83cmbR3E_zP0zdQSpd-r2-8:-N87xEBF445uh16wLDZP:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/98c6fcc6f43b4bc2e27be88bffb085d4d9f14533/878-ng-common-utilities/src/main/java/io/harness/gitaware/helper/GitImportInfoDTO.java#L8)):*
<details><summary><i>Previously, namanvermah (Naman Verma) wrote…</i></summary><blockquote>

We should move all the dtos that were in the same package as import info dto
</blockquote></details>

Done.

**Review by archit-harness — APPROVED** [review]
> Reviewed 32 of 32 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: 37 file(s) to review. 1 change requests. 1. unresolved discussions. 1 of 2 approvals obtained.

**Review by namanvermah — APPROVED** [review]
> Reviewed 1 of 6 files at r1, 4 of 8 files at r2, 32 of 32 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/35825)* status: :shipit: complete! ✓

**Pull request merged**